### ISSUE TITLE
Feature request: Support specifying vendor prefixes with property sort order

### ISSUE THREAD

**Issue by kristerkari** [brigade/scss-lint#511]
I looks like scss-lint ignores vendor prefixes if they're specified in the `order` config for `PropertySortOrder` linter.

Here is a failing test that I wrote (I'm not sure if it is correct, I tried to write it the same way as the other tests):

```ruby
  context 'when the order includes vendor prefixes' do
    let(:linter_config) { {
      'order' => %w["-ms-animation-delay" "-webkit-animation-delay" "-moz-animation-delay" "animation-delay"]
    } }

    context 'and the properties match the specified order' do
      let(:scss) { <<-SCSS }
        p {
          -ms-animation-delay: 1s;
          -webkit-animation-delay: 1s;
          -moz-animation-delay: 1s;
          animation-delay: 1s;
        }
      SCSS

      it { should_not report_lint }
    end

    context 'and the properties do not match the specified order' do
      let(:scss) { <<-SCSS }
        p {
          -ms-animation-delay: 1s;
          -moz-animation-delay: 1s;
          -webkit-animation-delay: 1s;
          animation-delay: 1s;
        }
      SCSS

      it { should report_lint line: 3 }
    end

  end
```

**Comment by sds** [comment]
Hey @kristerkari,

We explicitly ignore vendor prefixes because it seems like overkill to specify their exact order. Being able to just write `animation-delay` in your order list, and automatically ordering the `-moz-*`, `-ms-*`, and `-webkit-*`, prefixes to be in alphabetical order saves you a lot of typing in the `order` list.

Is there a strong reason you need to order the vendor prefixes a certain way?

**Comment by kristerkari** [comment]
> Is there a strong reason you need to order the vendor prefixes a certain way?

I was converting [existing configuration](https://github.com/csscomb/csscomb.js/blob/master/config/zen.json) from csscomb -> scss-lint, and it includes prefixes. The problem is that csscomb will order the prefixes in the specified order, but then scss-lint will give linting errors if the order is not what it expects (it ignores the prefix ordering that csscomb uses).

So, having this feature would make scss-lint work correctly when existing csscomb configuration is used.

**Comment by sds** [comment]
Thanks for clarifying, @kristerkari. Happy to accept a pull request fixing this.

**Comment by gunnx** [comment]
+1 for this, I use Sort lines function in IDE on properties and it puts vendor prefixes to the top which triggers the lint error.
